read-fonts/src/tables/aat.rs

```diff
@@ -218,8 +250,15 @@ pub type LookupU16<'a> = TypedLookup<'a, u16>;
 pub type LookupU32<'a> = TypedLookup<'a, u32>;
 pub type LookupGlyphId<'a> = TypedLookup<'a, GlyphId>;
 
+#[derive(Copy, Clone, bytemuck::AnyBitPattern)]
+pub struct NoPayload(());
```

Comment:
I'd like to at least document why this exists, we're going to have at least one confused reader somewhere down the road